checks.d/mysql.py

```diff
@@ -97,8 +97,8 @@ def _get_config(self, instance):
 
     def _connect(self, host, port, mysql_sock, user, password, defaults_file):
         service_check_tags = [
-            'host:%s' % host,
-            'port:%s' % port
+            'host:%s' % mysql_sock if mysql_sock != '' else host,
```

Comment:
what was it before ? i guess it was None or the empty string, right ? in that case localhost makes more sense than that